src/core/qgsdistancearea.cpp

```diff
@@ -537,6 +540,111 @@ double QgsDistanceArea::measureLine( const QgsPoint &p1, const QgsPoint &p2, Qgs
   return result;
 }
 
+double QgsDistanceArea::measureLineProjected( const QgsPoint &p1, double distance, double azimuth, QgsPoint *projected_point )
+{
+  double result = 0.0;
+  QgsPoint p2;
+  if ( geographic() )
```

Comment:
This is a problem.

The setting of 'mSemiMajor, mSemiMinor, mInvFlattening' must be done for this to work.

The default constructor is Wsg84, but not initialized.
Also for other functions such as setSourceCrs, these values are not being set when geographic.
* the question is why?